mappings/net/minecraft/entity/damage/DamageSource.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_1282 net/minecraft/entity/damage/DamageSource
 	FIELD field_28100 fallingBlock Z
 	FIELD field_28399 fromFalling Z
 	FIELD field_28400 STALAGMITE Lnet/minecraft/class_1282;
+	FIELD field_34004 noRetaliation Z
```

Comment:
Probably better to fit in with the `Angerable` interface, unless this field is more specific.

```suggestion
	FIELD field_34004 angering Z
```

Reply:
false means it should anger and true means it shouldn't.

Plus, this is independent of Angerable. Angerable is only implemented for mobs that are neutral; this affects any mob, like a Skeleton.

The code simply does not `setAttacker` when it `shouldNotRetaliate`. (That also means that if goat rams a dog, the poor dog's death message will not show the goat, if i'm correct.)